community/contributing/nightly-builds/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Nightly Builds
+layout: single
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+Bugs affecting more than one service commonly surface only when the whole system is running. Our continuous integration system builds and runs a suite of integration tests against Spinnaker nightly on real cloud provider infrastructure to detect these bugs.
+
+> **Access the CI system here**: [https://builds.spinnaker.io](https://builds.spinnaker.io)
+>
+> Requires membership in the `build-cops` GitHub Team to access the system.
+
+## Build Cop
+
+The **build cop** responsibilities include:
+
+* Triage integration test failures on the `master` and 3 most recent release branches
+* Clean up orphaned resources across target cloud providers
+* Route [new GitHub issues](https://github.com/spinnaker/spinnaker/issues) to the appropriate SIG
+  (applying GitHub labels as appropriate)
+* Observe any systemic problems raised in the [#general](https://spinnakerteam.slack.com/messages/general/)
+  and [#dev](https://spinnakerteam.slack.com/messages/dev/) Slack channels
```

Comment:
One additional task is the logging or creating of a record as to what occurred during a build rotation and how it was resolved. Is that being moved away from as part of these changes?

Reply:
Added. I initially omitted this, but thinking more about it, it will probably become more useful across organizational boundaries.